lib/known_error.ml

```diff
@@ -1,7 +1,7 @@
 open Base
 open Common
 
-type type_error = {filename: string; line: int; actual : Type.t; expected: Type.t; message: string}
+type type_error = {filename: string; expr: Ast.t; actual : Type.t; expected: Type.t; message: string}
```

Comment:
I think that `type_error` should have `line` (or related information) instead of `expr` .

Reply:
Thanks I'll fix it.

Reply:
Fixed